Cargo.toml

```diff
@@ -1,9 +1,48 @@
 [workspace]
-members = ["agent", "client", "crypto-auditing", "event-broker", "log-parser"]
+members = [
+    "agent",
+    "agent/tests/agenttest",
+    "client",
+    "crypto-auditing",
+    "event-broker",
+    "log-parser"
+]
 resolver = "2"
```

Comment:
(I know it's not changed in here, but I'd still like to ask)

> Features for target-specific dependencies are not enabled if the target is not currently being built.

is either a rather confusing feature description or a rather confusing feature. My testing seems to support the idea that the enabled feature set becomes the superset of the packages' features of the invocation. That suggests that 1. `cargo build` result would differ from `make`, 2. library featureset depends on whichever application target was built first, etc.

a. Should we strive for more isolation, e.g., by building the library first in a separate invocation?
b. Given that it still doesn't guarantee we've specified the features correctly, should we, maybe, give up and define just the all-encompassing superset at the top level and at least have the consistency?

Reply:
I'm not sure if I get it right, but I can for example turn off io_uring in the agent while keeping libsystemd in the event-broker with `make RELEASE=1 CARGO_ARGS="--no-default-features --features=libsystemd"`. Isn't sufficient?

Reply:
(clarified off-PR. We don't have builds that go "one app + library" and library inheriting a featureset of whatever app is built first, as I've mistakenly thought after seeing four app targets in Makefile. We'll either build the entire workspace at once with the union-of-all featureset or build subprojects one-by-one with their individual featuresets. As long as we test all of the approaches we use, this is fine.)